File: javascript/1131.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
const lines = input.split("\n");
var [values,a,b,inter,gremio,empates,grenais] = [0,0,0,0,0,0,0];
var escolha=0;

while(escolha !=2){
    console.log("Novo grenal (1-sim 2-nao)");

    var values = lines.shift().trim().split(" ");;
    a = parseInt(values.shift());
    b = parseInt(values.shift());
    //var maior = Math.max(a,b);
    if(a > b){
        inter++;
    }else if(b > a){
        gremio++;
    }else if(a == b){
        empates++;
    }
    grenais++;

    escolha=parseInt(lines.shift());
}

console.log(`${grenais} grenais`);
console.log(`Inter:${inter}`);
console.log(`Gremio:${gremio}`);
console.log(`Empates:${empates}`);

if(inter > gremio){
    console.log("Inter venceu mais");
}else if(gremio > inter){
    console.log("Gremio venceu mais");
}else{
    console.log("Nao houve vencedor");
}

File: C++/1131.cpp
#include <iostream>

using namespace std;

int main(){
    
    int a = 0, b = 0, inter = 0, gremio = 0, empates = 0, grenais = 0, escolha = 0;
    
    while(escolha != 2){
        cout << "Novo grenal (1-sim 2-nao)" << endl;
        cin >> a >> b;
        
        if(a > b){
            inter++;
        }else if(b > a){
            gremio++;
        }else if(a == b){
            empates++;
        }
        grenais++;
        
        cin >> escolha;
    }
    
    cout << grenais << " grenais" << endl;
    cout << "Inter:" << inter << endl;
    cout << "Gremio:" << gremio << endl;
    cout << "Empates:" << empates << endl;
    
    if(inter > gremio){
        cout << "Inter venceu mais" << endl;
    }else if (gremio > inter){
        cout << "Gremio venceu mais" << endl;
    }else {
        cout << "Nao houve vencedor" << endl;
    }
    return 0;
}